app/managers.py

```diff
@@ -0,0 +1,36 @@
+import sqlite3
+
+from app.models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect("cinema.db3")
+
+    def all(self) -> None:
+        actors_cursor = self._connection.execute(
+            "SELECT id, first_name, last_name FROM actors"
```

Comment:
You can use * to select all the columns from the table.

It is better to create an attribute to store the table name. By defining the table name as an attribute, you make the code more flexible and easier to update if the table name ever needs to be changed.